home-work13/txt_proxt_writter.py

```diff
@@ -0,0 +1,12 @@
+class TxtWriterProxy:
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.data = []
+
+    def write_data(self, record):
+        self.data.append(record)
+
+    def save_data(self):
+        with open(self.file_path, 'w') as file:
+            for record in self.data:
+                file.write(record + '\n')
```

Comment:
No. You should think about situation where is no need for writing to file.
In you case you write all the time